modules/021-kube-proxy/images/kube-proxy/werf.inc.yaml

```diff
@@ -1,24 +1,45 @@
+artifact: distroless-kube-proxy-artifact
```

Comment:
Traditionally we named artifacts as all another images with module path - {{ .ModuleName }}/distroless-kube-proxy-artifact

Reply:
Fixed in both `werf.imc.yaml`s.